spanner/tests/unit/test_keyset.py

```diff
@@ -26,6 +29,15 @@ def _getTargetClass(self):
     def _make_one(self, *args, **kwargs):
         return self._getTargetClass()(*args, **kwargs)
 
+    def _check_unused_keys(self, krange, used_keys):
```

Comment:
1. s/`krange`/`key_range` please
1. You should ditch this helper and just create the protobufs to compare against and really on the `__eq__` provided by the base `google.protobuf.Message` type